src/helpers/verifyAddUserForm.js

```diff
@@ -0,0 +1,29 @@
+import { displayErrorMessagesAddUSer } from '../helpers/displayErrorMessagesAddUser'
+import { errors } from '../helpers/errorMessages'
+import { displayErrorFor } from './errorMessageHandle'
+import users from '../data/users.json'
+
+export const verifyAddUserForm = (dispatchGetFormData, dispatchGetFormError, dispatchGetDoubleUsers, formData) => {
+
+  const allFieldsFilledIn = formData.name.length > 0 &&
+    formData.lastName.length > 0 &&
+    formData.email.length > 0 &&
+    formData.phone.length > 0
+
+  const emailValidationRegexp = /^([\w.%+-]+)@([\w-]+\.)+([\w]{2,})$/gi
+  const emailValidation = formData.email.match(emailValidationRegexp)
+
+  const phoneVelidationRegexp = /[0-9]+$/gi
+  const phoneValidation = formData.phone.match(phoneVelidationRegexp)
+
+  const doubleUserValidation = users.filter(user => user.email === formData.email)
+
+  const doubleUserValidationInput = dispatchGetDoubleUsers.length !== undefined ? dispatchGetDoubleUsers.filter(user => user.email === formData.email) : 0
```

Comment:
To co wcześniej - porównywanie do niespełnionego warunku. I parę innych rzeczy do przegadania ;)